pkgs/tools/text/ruby-zoom/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv {
+  name = "ruby-zoom-4.1.0";
+
+  inherit ruby;
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
+
+  meta = with lib; {
+    description = "Quickly open CLI search results in your favorite editor!";
+    homepage    = https://gitlab.com/mjwhitta/zoom;
+    license     = with licenses; gpl3;
+    platforms   = platforms.unix;
```

Comment:
This is how it usually works. I do not how passing dependencies to rubygems works in nixos.